### PR TITLE
Feature/profile page

### PR DESCRIPTION
**Pull request by Nekocut**

### CODE REVIEW

File: src/style/profile.less [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    max-width: 100px;
+    margin-top: 50px;
+    margin-bottom: 10px;
+    justify-content: center
+}
+
+.ProfilName{
+    background-color: #007EA7;
+    color: #f9fdff;
+    margin-top: 10px;
```
Comment (ermoca) [inline src/style/profile.less:12]: you can use the short version margin : top right bottom left (ex margin : 10px, auto, 80px, auto)

File: src/views/Profile.vue [hunk 2]
```diff
@@ -1,17 +1,58 @@
 <template>
   <div id="profile_page" :class="this.$store.state.css">
-    <h1>Profil utilisateur</h1>
-    <h3>Cette page est encore en construction</h3>
-    <p>Si vous avez été redirigé automatiquement ici, c'est que votre abonnement n'est pas à jour</p>
-    <p><a class="teamSelector" v-on:click="goToBillingPortal">{{$t('BILLING_MANAGEMENT')}}</a></p>
+    <div class="card-container">
+      <div class="upper-container"></div>
```
Comment (ermoca) [inline src/views/Profile.vue:4]: what does upper container do ?

File: src/views/Profile.vue [hunk 3]
```diff
@@ -1,17 +1,58 @@
...
+        <div>
+          <header>
+          <li>
+            <ul>
+              <a class="btnprofil fas fa-credit-card" v-on:click="goToBillingPortal">{{
+                $t("SUBCRIPTION")
+              }}</a>
+            </ul>
+            <ul>
+              <a class="btnprofil fas fa-book" v-on:click="goToBillingPortal">{{
```
Comment (ermoca) [inline src/views/Profile.vue:20]: you should remove the onclick function since the real one isn't already implemented. A nice way to do it would be to disable the button with a tooltip stating that it is a WIP

Comment (ermoca) [reply src/views/Profile.vue:20]: (same for all button except subscription and logout)

File: src/views/Profile.vue [hunk 4]
```diff
@@ -1,17 +1,58 @@
...
+                $t("APPEARANCE")
+              }}</a>
+            </ul>
+            <ul>
+              <a class="btnprofil fab fa-discord"  v-on:click="goToBillingPortal">{{
+                $t("COMUNITY")
+              }}</a>
+            </ul>
+            <ul>
+              <a class="btnprofil fas fa-power-off" v-on:click="login">{{
```
Comment (ermoca) [inline src/views/Profile.vue:35]: it should be logout instead of login

File: src/views/Profile.vue [hunk 5]
```diff
@@ -25,8 +66,10 @@ export default defineComponent({
     },
     logout () {
       PersonService.deleteSession(this.$store)
+    },
+    goChooseAccount () {
```
Comment (ermoca) [inline src/views/Profile.vue:70]: why did you add this function ?

**Review by ermoca — COMMENTED** [review]
Nice dive into the world of vue and css

**Comment by ermoca** [comment]
Some bug were identified while testing on my machine + the design is nice but still not consistent with the rest of the app. 
I think it would be easier to duplicate the existing pageToAccount.vue and replacing the current list of account by a list of the possible action. Will discuss it tomorrow together

File: src/views/Profile.vue [hunk 6]
```diff
@@ -1,32 +1,64 @@
...
-    <p>Si vous avez été redirigé automatiquement ici, c'est que votre abonnement n'est pas à jour</p>
-    <p><a class="teamSelector" v-on:click="goToBillingPortal">{{$t('BILLING_MANAGEMENT')}}</a></p>
-    <NavMenu :page="'profile'" />
+  <div :class="this.$store.state.css">
+    <div class="profilpage">
+      <div class="col-12 offset-0 col-sm-8 offset-sm-2 col-md-6 offset-md-3 col-lg-4 offset-lg-4 col-xxl-3 offset-xxl-5 ">
+        <div class="image-container">
+           <img id="logoface" alt="Vue logoface" src="../assets/logo_round.png" />
+        </div>
+         <h3 class="ProfilName">Nom Prenom</h3>
```
Comment (Clr34) [inline src/views/Profile.vue:8]: classes' names are in english in all the code and profile have an -e in english. Furthermore, they are written in camelCase : so line 3 profilpage become profilePage and line 8 ProfilName become profileName.
You can check all your classes and id to rectify their names and have a continuity in all code :)

File: src/views/Profile.vue [hunk 7]
```diff
@@ -1,32 +1,64 @@
...
+              }}</a>
+            </ul>
+            <ul>
+              <a class="buttonprofile"  v-on:click="disabled"><div class="icone fab fa-discord"></div>{{
+                $t("COMUNITY")
+              }}</a>
+            </ul>
+            <ul>
+              <a class="buttonprofile" v-on:click="logout"><div class="icone fas fa-power-off"></div>{{
+                $t(id="LOGOUT")
```
Comment (Clr34) [inline src/views/Profile.vue:32]: please remove "id=" from parenthesis, it's useless.

File: src/views/Profile.vue [hunk 8]
```diff
@@ -1,32 +1,64 @@
...
+    NavMenu
   },
   methods: {
-    async goToBillingPortal (): Promise<void> {
-      PersonService.redirectToBillingPortalUrl()
-    },
     logout () {
       PersonService.deleteSession(this.$store)
+    },
+    goToProfile () {
```
Comment (Clr34) [inline src/views/Profile.vue:59]: Where this method it called ? I don't see where...

File: src/views/Profile.vue [hunk 9]
```diff
@@ -1,32 +1,64 @@
...
+  <div :class="this.$store.state.css">
+    <div class="profilpage">
+      <div class="col-12 offset-0 col-sm-8 offset-sm-2 col-md-6 offset-md-3 col-lg-4 offset-lg-4 col-xxl-3 offset-xxl-5 ">
+        <div class="image-container">
+           <img id="logoface" alt="Vue logoface" src="../assets/logo_round.png" />
+        </div>
+         <h3 class="ProfilName">Nom Prenom</h3>
+        <li>
+            <ul>
+              <a class="buttonprofile" v-on:click="disabled"><div class="icone fas fa-credit-card"></div>{{
```
Comment (Clr34) [inline src/views/Profile.vue:11]: Like that this button call a method called disabled so it doesn't work because this method doesn't exist but for user buttons seem clickable, maybe write disabled in class (an object can have several class' names) and add a particular CSS for these objects this a class disabled. (you can inspire from NavigationMenu.vue and its CSS ButtonMenu.less)
the goal is to make the application the more user-friendly possible and learn :)

File: src/style/profile.less [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+.icone{
+    position: absolute;
+    margin-top: 3px;
+    margin-right: 200px;
+}
+
+.navmenu{
+    position: absolute;
+    margin-top: 20px;
+    margin-left: 30px;
```
Comment (Clr34) [inline src/style/profile.less:68]: navMenu is displaced compared to the white block, can you repair that ?
